Remotion/ObjectBinding/Web.ClientScript/Scripts/BocAutoCompleteReferenceValue.UI.ts

```diff
@@ -533,9 +533,11 @@ namespace Remotion.BocAutoCompleteReferenceValue
                     updateResult ({ DisplayName : input.value, UniqueIdentifier : options.nullValue });
                 } else {
                     clearTimeout(timeout);
-                    const lastKeyPressCode = state.lastKeyPressCode;
-                    invalidateResult();
-                    acceptInput(lastKeyPressCode, focusInputAfterSelection);
+                    if (state.previousValue !== input.value) {
+                        const lastKeyPressCode = state.lastKeyPressCode;
+                        invalidateResult();
```

Comment:
Can you please check if skipping only `invalidateResult()` would also be sufficient?

Reply:
Seems to work as well